
class RequestException(Exception):
    pass


class MainFunctionException(Exception):
    pass
